src/main/webapp/js/instructor.js

```diff
@@ -332,4 +332,14 @@ function updateHoverShowPictureEvents(actualLink, resolvedLink) {
         children('img[src=""]').attr('src', resolvedLink);
 }
 
+function toggleCommentsForStudents(sender){
+    var panelBody = $(sender).parent().find('div[class^="panel-body"]');
+    if(panelBody.hasClass('hidden')){
```

Comment:
Rather than having this conditional, perhaps you could use the jquery method toggleClass() or even simply just use toggle() to show/hide the panel body